example/src/App.js

```diff
@@ -518,6 +530,34 @@ const App = () => {
         </Select.WithRef>{' '}
         (required)
       </Section>
+      <Section title="DatePicker">
+        <DatePicker
+          showOutsideDays
+          onDayClick={handleDateClick}
+          selectedDays={selectedDay}
+        />
+        <br />
+        <Input
+          value={selectedDay.toISOString().split('T')[0]}
+          onClick={() => setShowInlineCalendar(true)}
+        />
+        {showInlineCalendar && (
+          <Paper
+            style={{
+              boxShadow: '0px 0px 2px 2px rgb(0 0 0 / 20%)',
+              display: 'table',
+              padding: '2px',
+              position: 'absolute',
+            }}
+          >
+            <DatePicker
+              showOutsideDays
+              onDayClick={handleInlineDateClick}
+              selectedDays={selectedDay}
+            />
+          </Paper>
+        )}
```

Comment:
This is an example of what it might look like to extend the DatePicker component to make it inline.